### PR TITLE
Fixes #21

### LINKED ISSUE #21: oca-update-pre-commit-excluded-addons not running on addon flipping from installable false to true
https://github.com/OCA/maintainer-tools/pull/474

### PR DESCRIPTION
**Pull request by sbidoul**

### CODE REVIEW

**Comment by sbidoul** [comment]
@Yajo looks like poetry install is acting up again ?

File: .pre-commit-config.yaml.jinja [hunk 1]
```diff
@@ -52,7 +52,7 @@ repos:
         language: fail
         files: "\\.rej$"
   - repo: https://github.com/oca/maintainer-tools
-    rev: 2c66e72
+    rev: fd75d80
```
Comment (Yajo) [inline .pre-commit-config.yaml.jinja:55]: Wouldn't it be better to do tag releases like everyone else? :thinking: 

That would allow autoupdating it.

**Comment by Yajo** [comment]
> looks like poetry install is acting up again ?

Let me check that

**Comment by Yajo** [comment]
Please rebase now that #28 is merged.

**Review by Yajo — APPROVED** [review]

File: .pre-commit-config.yaml.jinja (hunk 1 above)
Comment (sbidoul) [reply .pre-commit-config.yaml.jinja:55]: We could do maintainer-tools tag, yes.

**Pull request merged**